packages/block-library/src/breadcrumbs/index.php

```diff
@@ -448,21 +448,69 @@ function block_core_breadcrumbs_get_terms_breadcrumbs( $post_id, $post_type ) {
 		return array();
 	}
 
-	// Find the first taxonomy that has terms assigned to this post.
+	/**
+	 * Filters the preferred taxonomy and term for breadcrumbs on a per-post-type basis.
+	 *
+	 * Allows developers to specify which taxonomy and term should be used in breadcrumbs
+	 * when a post has multiple taxonomies or terms assigned.
+	 *
+	 * The preferred taxonomy must have terms assigned to the post, otherwise falls back to the first available taxonomy.
+	 *
+	 * The preferred term must exist and be assigned to the post. If the post has only one term, it's used regardless of preference.
+	 * If the preferred term is not found, falls back to the first term.
```

Comment:
Really off-topic, but I wish we'd introduce a style guideline on comment length. (Me, I'm old school and still format mine at 79–80 chars depending on the place, but anything between 72 and 90-ish would make me happy.)